docs/README.md

```diff
@@ -30,3 +30,10 @@ Please consider donating if you think docsify is helpful to you or that my work
 ## Community
 
 Users and the development team are usually in the [Gitter chat room](https://gitter.im/docsifyjs/Lobby).
```

Comment:
Should we change `Gitter` to `Discord`?

Reply:
@sy-records  I remembered that @trusktr updated a new README.md, I thing it will be changed later.

Reply:
#1224